packages/patternfly-4/react-docs/package.json

```diff
@@ -24,14 +27,15 @@
     "prop-types": "^15.6.1",
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
+    "react-emotion": "^9.2.9",
     "react-helmet": "^5.2.0"
   },
   "keywords": [
     "gatsby"
   ],
   "scripts": {
-    "build": "gatsby build",
-    "postbuild": "node build/copyDocs.js",
+    "build": "yarn copy-docs && gatsby build",
```

Comment:
This was moved to post build since I believe it relys on react-core being finished

Reply:
I had moved it up here so it could copy some assets to the static folder then gatsby would copy that into the public folder during build, but i will move it back to postbuild and copy into the public folder instead